config/src/test/kotlin/org/springframework/security/config/web/server/ServerCsrfDslTests.kt

```diff
@@ -237,27 +238,29 @@ class ServerCsrfDslTests {
     @EnableWebFluxSecurity
     @EnableWebFlux
     open class MultipartFormDataNotEnabledConfig {
-        companion object {
-            var TOKEN_REPOSITORY: ServerCsrfTokenRepository = mock(ServerCsrfTokenRepository::class.java)
-        }
+
+        val serverCsrfTokenRepository: ServerCsrfTokenRepository = mockk()
```

Comment:
Since we can't use a mock or null here, we must create an actual instance.
```suggestion
       companion object {
            var TOKEN_REPOSITORY: ServerCsrfTokenRepository = WebSessionServerCsrfTokenRepository()
        }
```